brands/amenity/prep_school.json

```diff
@@ -0,0 +1,68 @@
+{
+  "amenity/prep_school|Huntington Learning Center": {
+    "countryCodes": ["us"],
+    "matchNames": ["huntington"],
+    "tags": {
+      "amenity": "prep_school",
+      "brand": "Huntington Learning Center",
+      "brand:wikidata": "Q5945399",
+      "brand:wikipedia": "en:Huntington Learning Center",
+      "name": "Huntington Learning Center"
+    }
+  },
+  "amenity/prep_school|KUMON (Japan)": {
+    "countryCodes": ["jp"],
+    "matchNames": ["くもん", "公文"],
+    "nomatch": ["amenity/prep_school|Kumon"],
+    "tags": {
+      "amenity": "prep_school",
+      "brand": "KUMON",
+      "brand:wikidata": "Q142054",
+      "brand:wikipedia": "ja:日本公文教育研究会",
+      "name": "KUMON",
+      "name:ja": "公文",
+      "name:ja-Hira": "くもん",
+      "name:ja-Latn": "KUMON"
```

Comment:
OSM has features named all three ways in Japan, but as far as I can tell, the branding is always the same.